File: src/01-easy/1d-bush-fire/solution.php
<?php
$n = intval(trim(fgets(STDIN)));
for ($i = 0; $i < $n; $i++) {
    $s = trim(fgets(STDIN)); $d = 0; $j = 0;
    while ($j < strlen($s)) { if ($s[$j] === 'f') { $d++; $j += 3; } else $j++; }
    echo "$d\n";
}

File: src/01-easy/1d-bush-fire/solution.c
#include <stdio.h>
#include <string.h>
int main(){
    int n; scanf("%d",&n);
    while(n--){
        char s[256]; scanf("%s",s);
        int d=0,j=0,l=strlen(s);
        while(j<l){if(s[j]=='f'){d++;j+=3;}else j++;}
        printf("%d\n",d);
    }
}
